.test-defs/cmd/helper.go

```diff
@@ -12,16 +12,18 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package main
+package cmd
 
 import (
+	"github.com/sirupsen/logrus"
+
 	gardenv1beta1 "github.com/gardener/gardener/pkg/apis/garden/v1beta1"
 	"github.com/gardener/gardener/pkg/operation/common"
 )
 
-// updateWorkerZone updates the zone of the workers.
+// UpdateWorkerZone updates the zone of the workers.
 // Azure shoots are ignored.
-func updateWorkerZone(shoot *gardenv1beta1.Shoot, cloudprovider gardenv1beta1.CloudProvider, zone string) {
+func UpdateWorkerZone(testLogger *logrus.Logger, shoot *gardenv1beta1.Shoot, cloudprovider gardenv1beta1.CloudProvider, zone string) {
```

Comment:
Why are you passing in the `*logrus.Logger` in the functions now? I guess it's not needed.